t/t0012-content-sqlite.t

```diff
@@ -134,7 +134,7 @@ test_expect_success 'checkpoint-put w/ rootref bar' '
 
 test_expect_success 'checkpoint-get returned rootref bar' '
 	echo bar >rootref.exp &&
-	checkpoint_get | jq -r .value | jq -r .rootref >rootref.out &&
+	checkpoint_get | jq -r .value[0] | jq -r .rootref >rootref.out &&
```

Comment:
Maybe for later cleanup, but these jq invocations could easily be combined, e.g.
```
checkpoint_get | jq -r .value[0].rootref
```
Two potential gotchas with the way these tests are structured:
-  I think without `-e`, jq returns success even if the requested key/array entry is not found
- A pipeline succeeds if the last command in in succeeds

Not saying these tests are broken but they might be a little fragile as is.

Reply:
it seems we do the double `jq` elsewhere as well.  I'll do a cleanup PR after this PR.